jonathangrm-belfast.md

```diff
@@ -0,0 +1,6 @@
+# Portstewart, Northern Ireland
+
+## Places to Eat
+- McDonalds not for :beer: :smile:
+- Tides
```

Comment:
I've never been :rofl: Although I hear the food tastes great. It also sits high up, overlooking the beach.